README.md

```diff
@@ -9,6 +9,12 @@ Most documentation for the service can be found on the
 Some app-specific technical documentation can be found in the [docs](docs)
 directory.
 
+### dxw support
```

Comment:
I'm not sure this needs to be "dxw" specifically. dxw have a support contract now, but that may not be the case in six months. Perhaps better to keep this generic, e.g. "Developer first-line support" so it would equally apply to say a DfE support team that takes over support duties.